lib/gather/gather_pool.js

```diff
@@ -37,16 +37,16 @@ const GATHER_CONFIGS = [
 	},*/
 	{
 		type: "invitational",
-		icon: '/inviteGather.png',
-		name: "Invitational Gather",
-		description: "Join on ensl.org/teams/949",
-		// Grant invite if on list
-		membershipTest: function (user) {
-			return InvitationalGather.list.some(m => m.id === user.id);
-		},
-		serverMembershipTest: function (server) {
-			return server.name.toLowerCase().indexOf("promod") === -1;
-		}
+        icon: "/inviteGather.png",
+        name: "Invitational Gather",
+        description: "You must be on a nsl team to join",
+        // Grand invite if on any nsl team
+        membershipTest: function (user) {
+            return user.team ? true : false;
```

Comment:
using a ternary operator like this is not really good practice, although better than using double exclamation mark (!!) imho
what you should do here is explicitly checking for the falsy value, so in case this is undefined you shoud use
return user.team != undefined, not all to sure what the value here is though but i guess undefined is a safe guess